### PR TITLE
SMS Notification support for alerts using Twillio

### PR DESCRIPTION
**Pull request by SagarKarwande**
- SMS based support for alerts app

Following envirnoment variable need to be setup for SMS sending using twilio:
export TWILIO_ACCOUNT_SID
export TWILIO_AUTH_TOKEN
export TWILIO_NUMBER

### CODE REVIEW

File: apps/notifications/lib/notifications.ex [hunk 1]
```diff
@@ -32,6 +32,7 @@ defmodule Notifications do
   end
 
   defp send_sms(alert, vendor) do
+    twilio_no = mqtt_port = System.fetch_env!("TWILIO_NUMBER")
```
Comment (BandanaPandey) [inline apps/notifications/lib/notifications.ex:35]: @SagarKarwande Where are we using mqtt_port ?

File: apps/notifications/lib/notifications.ex [hunk 2]
```diff
@@ -59,22 +61,30 @@ defmodule Notifications do
...
 
   defp create_message(alert) do
+    parameter_text =
+      alert.alert_metadata
+      |> Enum.map(fn {key, value} -> "#{key}: #{value}" end)
+      |> Enum.join("\n")
+
     "Hello,
-    An alert has been generated in your MADS workspace on #{alert.inserted_at}.
+    An alert has been generated in your MADS workspace on #{alert.inserted_timestamp}.
```
Comment (BandanaPandey) [inline apps/notifications/lib/notifications.ex:77]: @SagarKarwande  here shouldn't we be using `inserted_at` instead of inserted_timestamp ?
Thoughts ?

**Review by BandanaPandey — CHANGES_REQUESTED** [review]

File: apps/notifications/lib/notifications.ex (hunk 1 above)
Comment (SagarKarwande) [reply apps/notifications/lib/notifications.ex:35]: it is a typo

Comment (SagarKarwande) [reply apps/notifications/lib/notifications.ex]: It is fixed

File: apps/notifications/lib/notifications.ex [hunk 3]
```diff
@@ -59,38 +61,58 @@ defmodule Notifications do
...
+
+    #{alert.message}
```
Comment (BandanaPandey) [inline apps/notifications/lib/notifications.ex:79]: @SagarKarwande  Can we make it conditional,
If alert.message is present than only show it, else not show it ?

**Review by BandanaPandey — APPROVED** [review]

**Pull request merged**